unique_toolkit/unique_toolkit/language_model/prompt.py

```diff
@@ -0,0 +1,45 @@
+from string import Template
+from typing import Any
+
+from unique_toolkit.language_model.schemas import (
+    LanguageModelSystemMessage,
+    LanguageModelUserMessage,
+)
+
+
+class Prompt:
```

Comment:
Can you add a possibility to see what are the required parameters of the template?

Reply:
Yes, I can add a function for that.

Reply:
as discussed, will be added once we need it